deploy/olm-catalog/knative-kafka-operator/0.12.1/knative-kafka-operator.v0.12.1.clusterserviceversion.yaml

```diff
@@ -60,285 +75,6 @@ spec:
   install:
     spec:
       clusterPermissions:
-      - rules:
```

Comment:
@aliok all the removed bits here, are those now actually taken into account from the "applied" yamls (of the `KafkaChannel`) ?

Reply:
yeah, the operator-sdk version I have I think has a bug that it considered everything under `/deploy` recursively and ended up having all the resources here in this file. 

These rules are actually for the things that are provisioned by the operator, not for the operator itself.